components/Banner/examples/supplemental/MDCBannerView.m

```diff
@@ -14,24 +14,40 @@
 
 #import "MDCBannerView.h"
 
-#import "MDCBannerViewLayout.h"
 #import "MaterialButtons.h"
+#import "MaterialTypography.h"
+
+typedef NS_ENUM(NSInteger, MDCBannerViewLayoutStyle) {
```

Comment:
Suggestion: `MDCBannerViewLayoutMode`

This doesn't particularly have to do with style.

Reply:
Thanks for the suggestion. Addressed in a34769e62030086202b79c041505a0dd813563f7.